### PR TITLE
186 modified createRecords method to check if the user has instituion

### PR DESCRIPTION
**Pull request by palagdan**
Partially resolves kbss-cvut/record-manager-ui/issues/186

### CODE REVIEW

File: src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java [hunk 1]
```diff
@@ -150,9 +155,15 @@ private PatientRecord findInternal(String key) {
...
 
+
+
     @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.CREATED)
-    public ResponseEntity<Void> createRecord(@RequestBody PatientRecord record) {
+    public ResponseEntity<String> createRecord(@RequestBody PatientRecord record) {
+
+        if(userService.getCurrentUser().getInstitution() == null)
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("User has no relation to the institution");
```
Comment (blcham) [inline src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java:165]: What do you think about 409 instead?

Comment (blcham) [reply src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java:165]: @palagdan

Comment (palagdan) [reply src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java:165]: Yes, I think that would be better

**Comment by palagdan** [comment]
I changed a description of the error to: "User is not assigned to any institution". 

After that frontend recieves this description and renders "Unable to create a record. { description from backend }

**Pull request merged**